src/style.css

```diff
@@ -1,3 +1,80 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+.logo a {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav ul {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav ul li {
+  color: black;
+}
+
+.nav ul li:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav ul li:hover {
+  color: #00acdc;
+}
+
+.nav ul li a {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin: auto;
+  text-transform: uppercase;
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 500;
+}
+
```

Comment:
try to avoid styling the tags like that, better to add some class for li (nav__item) and `a` (nav__link) and style it using these classes

Reply:
I fixed it